controllers/providers/dns/factory_test.go

```diff
@@ -33,7 +33,7 @@ import (
 
 func TestFactoryInfoblox(t *testing.T) {
 	// arrange
-	client := fake.NewFakeClientWithScheme(scheme.Scheme, []runtime.Object{}...)
+	client := fake.NewClientBuilder().WithScheme(scheme.Scheme).WithRuntimeObjects([]runtime.Object{}...).Build()
```

Comment:
Is it dictated by newer fake client version?

Reply:
It's deprecated. see upstrean function in controller runtime:  [uses NewClientBuilder anyway and  comment says it's deprecated](https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/client/fake/client.go#L73-L80)